libraries/c_sdk/standard/https/src/iot_https_client.c

```diff
@@ -2029,6 +2033,9 @@ static IotHttpsReturnCode_t _flushHttpsNetworkData( _httpsConnection_t * pHttpsC
     IotHttpsReturnCode_t networkStatus = IOT_HTTPS_OK;
     size_t numBytesRecv = 0;
 
+    /* Disable -Wunused-but-set-variable for local variables used for logging. */
+    ( void ) pHttpParserErrorDescription;
+
```

Comment:
Again this seems to be dead code that needs to be removed.

Reply:
Same for this as well. This variable is used for logging.